fastlane/lib/fastlane/actions/update_app_identifier.rb

```diff
@@ -47,6 +47,16 @@ def self.run(params)
         end
       end
 
+      def self.resolve_path(path, xcodeproj_path)
+        project_dir = File.dirname(xcodeproj_path)
+        # SRCROOT and PROJECT_DIR are the same
+        %w{SRCROOT PROJECT_DIR}.each do |variable_name|
```

Comment:
According to the official documentation.
There is also `SOURCE_ROOT` as an alias for `SRCROOT`.
https://developer.apple.com/legacy/library/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW105

Reply:
Didn't know about this one, done, thanks.